repose-aggregator/core/repose-core/src/main/scala/org/openrepose/powerfilter/ReposeFilter.scala

```diff
@@ -59,6 +60,7 @@ class ReposeFilter @Inject()(@Value(ReposeSpringProperties.NODE.NODE_ID) nodeId:
                              optMetricsService: Optional[MetricsService],
```

Comment:
Did we add a note to the Upgrade Guide that the PowerFilter was renamed to the ReposeFilter in case anyone has any custom logging for the specific class?

Reply:
We did not.

On it.